src/server/router.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+
+import userThemeRoutes from './routes/userThemeRoutes';
+
+const router: Router = Router();
```

Comment:
❗️
Тут тайпскрипт сам поймёт, что происходит)
```suggestion
const router = Router();
```

Reply:
🧐
В файле ниже предложил как лучше организовать роутеры